src/realm-settings/ProfilesTab.tsx

```diff
@@ -15,36 +15,51 @@ import { useTranslation } from "react-i18next";
 import { useAdminClient } from "../context/auth/AdminClient";
 import { useConfirmDialog } from "../components/confirm-dialog/ConfirmDialog";
 import { useRealm } from "../context/realm-context/RealmContext";
-import { cellWidth, IFormatterValueType } from "@patternfly/react-table";
 import { useAlerts } from "../components/alert/Alerts";
 import { Link } from "react-router-dom";
-import type ClientProfilesRepresentation from "@keycloak/keycloak-admin-client/lib/defs/clientProfilesRepresentation";
 import "./RealmSettingsSection.css";
+import type ClientPolicyExecutorRepresentation from "@keycloak/keycloak-admin-client/lib/defs/clientPolicyExecutorRepresentation";
+
+type ClientProfile = {
+  description?: string;
+  executors?: ClientPolicyExecutorRepresentation[];
+  name?: string;
+  global?: boolean;
```

Comment:
Looking at the code in the loader I think you can make this field required as it is always added to the object before `setProfiles()` is called.
```suggestion
  global: boolean;
```

Reply:
You're right! thanks @jonkoops